jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/quicknewjob/NewJobViewImpl.java

```diff
@@ -0,0 +1,463 @@
+    @DataField("job-name-group")
+    private FormGroup jobNameGroup;
+
+    @Inject
+    @DataField("job-name-help")
+    private Span jobNameHelp;
+
+    @Inject
+    @DataField("date-filters-input")
+    Div dateFiltersInput;
+
+    @Inject
+    @DataField("launch-now")
+    private RadioInput jobLaunchNowRadio;
+
+    @Inject
+    @DataField("job-type-label")
+    private FormLabel jobTypeLabel;
+
+    @Inject
+    @DataField("job-type-input")
+    private TextInput jobTypeInput;
+
+    @Inject
+    @DataField("job-type-group")
+    private FormGroup jobTypeGroup;
+
+    @Inject
+    @DataField("job-type-help")
+    private Span jobTypeHelp;
+
+    @Inject
+    @DataField("job-retries-label")
+    private FormLabel jobRetriesLabel;
+
+    @Inject
+    @DataField("job-retries-input")
+    private NumberInput jobRetriesInput;
+
+    @Inject
+    @DataField("job-retries-group")
+    private FormGroup jobRetriesGroup;
+
+    @Inject
+    @DataField("job-retries-help")
+    private Span jobRetriesHelp;
+
+    @Inject
+    @DataField("modal")
+    private Modal modal;
+
+    @Inject
+    @DataField("alert")
+    private InlineNotification inlineNotification;
+
+    @Inject
+    private ManagedInstance<DateRangePicker> dateRangePickerProvider;
+
+    private DateRangePicker dateRangePicker;
+
+    private Date selectedDate;
+
+    protected NewJobPresenter presenter;
+
+    public void init(NewJobPresenter presenter) {
+        this.presenter = presenter;
+    }
+
+    private ListDataProvider<RequestParameterSummary> dataProvider = new ListDataProvider<RequestParameterSummary>();
+
+    public DataGrid<RequestParameterSummary> myParametersGrid = new DataGrid<RequestParameterSummary>();
+
+    @PostConstruct
+    public void init() {
+        jobNameLabel.addRequiredIndicator();
+        jobTypeLabel.addRequiredIndicator();
+        jobRetriesLabel.addRequiredIndicator();
+
+        jobRetriesInput.setType("number");
+        jobRetriesInput.setAttribute("min",
+                                     "0");
+        jobRetriesInput.setDefaultValue("0");
+        jobRetriesInput.addEventListener("keypress",
+                                         getNumericInputListener(),
+                                         false);
+
+        myParametersGrid.setHeight("200px");
+        myParametersGrid.setEmptyTableWidget(new Label(constants.No_Parameters_added_yet()));
+
+        initGridColumns();
+        initDateTimePicker();
+
+        advancedContent.add(myParametersGrid);
+        inlineNotification.setType(InlineNotification.InlineNotificationType.DANGER);
+    }
+
+    protected EventListener<KeyboardEvent> getNumericInputListener() {
+        return (KeyboardEvent e) -> {
+            int keyCode = e.getKeyCode();
+            if (keyCode <= 0) { //getKeyCode() returns 0 for numbers on Firefox 53
+                keyCode = e.getWhich();
+            }
+            if (!((keyCode >= KeyCodes.KEY_NUM_ZERO && keyCode <= KeyCodes.KEY_NUM_NINE) ||
```

Comment:
Does this mean that we don't allow entering 0 retries?

Reply:
yeah, we allow keycode == KeyCodes.KEY_ZERO

Reply:
@bsremac Here I only add the same control that was used at advanced search. Tested on Chrome it works fine, allowing entering the 0 value. If you find it don't work in any case, we can review that